docs/Custom_Command_Keybindings.md

```diff
@@ -314,6 +315,11 @@ CheckedOutBranch
 
 To see what fields are available on e.g. the `SelectedFile`, see [here](https://github.com/jesseduffield/lazygit/blob/master/pkg/gui/services/custom_commands/models.go) (all the modelling lives in the same file).
 
+We don't support accessing all elements of a range selection yet; the properties above only give you the moving end of the range. We might add this in the future, but as a special case you can access the range of selected commits by using `SelectedCommitRange`, which has two properties `.To` and `.From` which are the hashes of the bottom and top selected commits, respectively. This is useful for passing them to a git command that operates on a range of commits. For example, to create patches for all selected commits, you might use
```

Comment:
This part:
> ; the properties above only give you the moving end of the range

Confused me for a bit because I didn't know what it meant. I don't think this part is necessary: it's sufficient to just say we don't support range selection.

Reply:
I thought it would be good to explain what happens if you access the single-element properties while there's a range selection, since I don't find this entirely obvious. And "moving end of the range" is the only way I could come up with to describe this.

But apparently it didn't work, so I'm happy to take this out if it's more confusing than helpful (9e5741898c42).

Feel free to push a fixup if you want to improve the text further, I'll leave the PR open for a day or two more.

Reply:
> I thought it would be good to explain what happens if you access the single-element properties while there's a range selection, since I don't find this entirely obvious. And "moving end of the range" is the only way I could come up with to describe this.

I think that's an edge case we don't need to worry about.

Code LGTM so feel free to merge